router/middleware/session/repo.go

```diff
@@ -157,11 +157,13 @@ func MustPull(c *gin.Context) {
 
 	// debugging
 	if user != nil {
-		c.AbortWithStatus(http.StatusNotFound)
+		c.String(http.StatusNotFound, http.StatusText(http.StatusNotFound))
```

Comment:
I don't think that this error message is really helpful. And supplies for the messages below